content/post/084-explore-the-anatomy-of-a-cid-on-protoschool.md

```diff
@@ -0,0 +1,54 @@
+---
+date: 2020-03-04
+url: 2020-03-04-explore-the-anatomy-of-a-cid-on-protoschool
+title: Explore the Anatomy of a CID on ProtoSchool
+author: José Bateira and Teri Chadbourne
+---
+The ProtoSchool team is pleased to announce the launch of a shiny new multiple-choice tutorial on the **[Anatomy of a CID](https://proto.school/#/anatomy-of-a-cid)**, built by the fabulous [José Bateira](https://github.com/zebateira)! It's our first multiple-choice lesson and the first to explore Multiformats. 🎉
+
+![Tutorial Table of Contents](/img/084-explore-the-anatomy-of-a-cid-on-protoschool/table-of-contents.png)
+![Sample Lesson](/img/084-explore-the-anatomy-of-a-cid-on-protoschool/sample-lesson.png)
+
+Content identifiers (CIDs) and cryptographic hashing make content addressing possible, allowing us to securely locate and identify data on the decentralized web. The string representations of CIDs used by IPFS look like random characters strung together, but there's a lot more going on under the hood. In this new tutorial we explore all of the data revealed by these self-describing content-addressed identifiers, digging deep into multihash, multicodec, multibase, and version prefixes.
+
+We also take a look at the evolution of the [CID specification](https://github.com/multiformats/cid), which originated in IPFS and now lives in the [Multiformats Project](https://multiformats.io/), a collection of self-describing protocols that future-proof distributed information systems including IPFS, IPLD, libp2p, and Filecoin. Curious why some IPFS CIDs start with `Qm...` and others start with `b`? Wondering why we made the switch from CIDv0 to CIDv1, or whether those versions are interchangeable? We've got you covered!
```

Comment:
```suggestion
We also take a look at the evolution of the [CID specification](https://github.com/multiformats/cid), a collection of self-describing protocols that future-proof distributed information systems including IPFS, IPLD, libp2p, and Filecoin. Curious why some IPFS CIDs start with `Qm...` and others start with `b`? Wondering why we made the switch from CIDv0 to CIDv1, or whether those versions are interchangeable? We've got you covered!
```

This sentence is already pretty lengthy. Mentioning where the CID spec lives doesn't add much value _right here_.

Reply:
Ah, your update doesn't work because "a collection of self-describing protocols that future-proof distributed information systems including IPFS, IPLD, libp2p, and Filecoin" describes Multiformats, not the CID spec. Is there a better spot you'd suggest to call out the Multiformats relationship?

Reply:
How about:

```markdown
We also take a look at the evolution of the [CID specification](https://github.com/multiformats/cid), which originated in IPFS and now lives in the [Multiformats Project](https://multiformats.io/). The Multiformats Project is a collection of self-describing protocols that future-proof distributed information systems including IPFS, IPLD, libp2p, and Filecoin. Curious why some IPFS CIDs start with `Qm...` and others start with `b`? Wondering why we made the switch from CIDv0 to CIDv1, or whether those versions are interchangeable? We've got you covered!
```

Reply:
Sure, happy to break it up. Thanks for the input!